scrunch/datasets.py

```diff
@@ -2013,6 +2019,8 @@ def forks_dataframe(self):
             A DataFrame representation of all attributes from all forks
             on the given dataset.
         """
+        if pd is None:
+            raise ModuleNotFoundError("No module named 'pandas'")
```

Comment:
This message should read more like "Pandas is not installed, please install it in your environment to use this function" or something along those lines.

Reply:
I was thinking on replicate the error that python raises when you import something that is not installed, but your message makes more sense